product_docs/docs/pgd/5/admin-manual/installing/07-configure-proxies.mdx

```diff
@@ -102,6 +102,16 @@ CREATE USER pgdproxy PASSWORD 'proxysecret';
 GRANT bdr_superuser TO pgdproxy;
 ```
 
+Set node level route_dsn option for each node with database user pgdproxy. The proxy uses these dsn while connecting to database nodes. Please note the initial dsn (on proxy start) is taken from proxy config file (see below). So, ideally, route_dsn given in this section and in config file should match.
```

Comment:
General point: Do not use relative directions like (see below) in a sequential tutorial or example.

Reply:
I'm not sure I get it but removing `(see below)`. Quick search suggest 12 places where `(see below)` is mentioned in docs repo.

Reply:
Yes, and our style guide says don't do it, and there's a lot of pre-style guide content in the PGD docs which is yet to be tidied up.